<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sesion</class>
 <widget class="QMainWindow" name="Sesion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>NSimSun</family>
    <pointsize>18</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="grilla">
   <widget class="QGroupBox" name="box">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>532</width>
      <height>282</height>
     </rect>
    </property>
    <property name="title">
     <string>Ingreso a Sinergia</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="msg">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>50</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="msg2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuario o correo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Contrase√±a:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ent">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>271</width>
       <height>35</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ent2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>271</width>
       <height>35</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Entrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Registrarse</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>M</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="msg5">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
